src/UI/Implementation/Component/Popover/Popover.php

```diff
@@ -0,0 +1,144 @@
+<?php
+namespace ILIAS\UI\Implementation\Component\Popover;
+
+use \ILIAS\UI\Component;
+use ILIAS\UI\Component\Signal;
+use ILIAS\UI\Implementation\Component\ComponentHelper;
+use ILIAS\UI\Implementation\Component\SignalGeneratorInterface;
+
+class Popover implements Component\Popover\Popover {
+
+	use ComponentHelper;
+
+	const POS_AUTO = 'auto';
+	const POS_VERTICAL = 'vertical';
+	const POS_HORIZONTAL = 'horizontal';
```

Comment:
Constants in interfaces: Do we want this? It seems bad practise. However, in this case these constants are part of the public interface, as you could pass them via `Popover::withPosition()`. We could also introduce public methods `withHorizontalPosition()` and `withVerticalPosition()`, then the constants would be implementation detail. What do you think? I guess it also depends on the number of possible positions we might offer.